web/crux-ui/src/validations/deployment.ts

```diff
@@ -18,5 +18,6 @@ export const createDeploymentSchema = updateDeploymentSchema.concat(
 export const copyDeploymentSchema = createDeploymentSchema
 
 export const createDeploymentTokenSchema = yup.object().shape({
+  name: yup.string().required(),
```

Comment:
We usually have a rule for names to be at least 3 characters, isn't that required here too?